### DISCUSSION TITLE
Welcome to iocage-homeassistant Discussions!

### DISCUSSION THREAD

**Discussion by tprelog** [tprelog/iocage-homeassistant#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adorobis** [comment]
Hi,
I'm Adam and using HA on FreeNAS (now on TrueNAS Core) since beginning of the year. Switched to the community plugin as soon as it appeared.
@tprelog many thanks for creating it and maintaining! Hope you'll continue even after you switch to the Scale edition so far I'm not planning to do so, don't have spare hardware to make any tests but could try to install the Scale edition on an alternative boot drive. Let's see when it is released. In your testing, do you see any major benefits compared to running HA in jail?
